src/test/groovy/pl/tomaszdziurko/jvm_bloggers/InitialBlogDataPopulationTriggerSpec.groovy

```diff
@@ -0,0 +1,42 @@
+package pl.tomaszdziurko.jvm_bloggers
+
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.BlogPostsFetchingScheduler;
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPostRepository
+import pl.tomaszdziurko.jvm_bloggers.blogs.BloggersDataFetchingScheduler;
+import pl.tomaszdziurko.jvm_bloggers.blogs.domain.BlogRepository;
+import spock.lang.Specification;
+import spock.lang.Subject
+
+class InitialBlogDataPopulationTriggerSpec extends Specification {
+
+    def BlogRepository blogRepository = Mock()
```

Comment:
Class name is redundant in this case. Mock type will be inferred from the variable type, so it should be either `BlogRepository blogRepository = Mock()` or `def blogRepository = Mock(BlogRepository)`. Please decide.